ami/client/flowchart.py

```diff
@@ -121,17 +121,17 @@ def __init__(self, proc, parent=None):
     def moveEvent(self, event):
         super().moveEvent(event)
         self.proc.node.geometry = self.saveGeometry()
-        self.proc.send_checkpoint(self.proc.node)
+        self.proc.send_checkpoint(self.proc.node, 'moveEvent')
```

Comment:
Why is the event type suddenly needed?

Reply:
It was useful for debugging in updateState() in the editor because I could set a breakpoint based on which function sent the checkpoint.